app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/SelectDutiesController.scala

```diff
@@ -43,6 +43,11 @@ import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 
 import scala.concurrent.ExecutionContext
 
+final case class CmaEligibleAndDuties(
+  isCmaEligible: List[Boolean],
```

Comment:
Optional: Perhaps Seq is better. As result you can either use Vector or List. Here are debate:
https://stackoverflow.com/questions/6928327/when-should-i-choose-vector-in-scala
Although if list is relatively small I reckon there is no significant difference. So it is optional.

Reply:
Done